core/domain/question_services.py

```diff
@@ -153,6 +153,55 @@ def get_question_by_id(question_id, strict=True):
         return None
 
 
+def get_question_skill_links_of_skill(skill_id):
+    """Returns a list of QuestionSkillLink domains of a particular skill ID.
+
+    Args:
+        skill_id: str. ID of the skill
+
+    Returns:
+        list(QuestionSkillLink)|None. The list of question skill link
+        domains that are linked to the skill ID. None if it doesnt exist.
+    """
+
+    question_skill_link_model_list = (
+        question_models.QuestionSkillLinkModel.get_question_skill_models_of_skill( #pylint: disable=line-too-long
+            skill_id)
+    )
+    question_skill_link_domains = [
+        question_domain.QuestionSkillLink(
+            question_skill_link_model.question_id,
+            question_skill_link_model.skill_id)
+        for question_skill_link_model in question_skill_link_model_list
+    ]
+
+    return question_skill_link_domains
+
+
+def _save_question_skill_link(question_skill_link, question_skill_model_id):
```

Comment:
SInce, you are only using question_skill_link.id here, maybe pass only that?
Also docstrings.

Reply:
That makes sense, changed it